.changeset/olive-glasses-bake.md

```diff
@@ -0,0 +1,7 @@
+---
+"next-example": patch
+"e2e": patch
```

Comment:
These need to be removed 🙃

Reply:
```suggestion
```

Reply:
im thinkg about a GitHub action checks your changeset doesn't have any private packages until https://github.com/changesets/bot/issues/44 is resolved.